frontend/src/scenes/web-analytics/WebAnalyticsScene.tsx

```diff
@@ -6,38 +6,38 @@ import { NodeKind } from '~/queries/schema'
 export function WebAnalyticsScene(): JSX.Element {
     return (
         <div>
```

Comment:
I'd suggest adding a `<div>` tag around all the titles, and replacing this line with

```suggestion
        <div className='space-y-2'>
```

or so. 

Check out the https://storybook.posthog.net/ for more style guidelines.

Reply:
I'm intentionally not including any styling changes in this PR - that'll be a separate PR. The text here is just for my benefit as I develop it.